Formula/libgr.rb

```diff
@@ -11,19 +11,22 @@ class Libgr < Formula
     sha256 "beaa733249692d24749156ad652ac3a3144cbd4e571e639c2088bb6331b1598b" => :high_sierra
   end
 
+  depends_on "cmake" => :build
   depends_on xcode: :build
   depends_on "cairo"
   depends_on "glfw"
   depends_on "libtiff"
+  depends_on "qhull"
   depends_on "qt"
   depends_on "zeromq"
 
   def install
-    # TODO: Remove this when released archive includes
-    # the fix of https://github.com/sciapp/gr/pull/101 .
-    ENV.deparallelize
-    system "make", "GRDIR=#{prefix}"
-    system "make", "GRDIR=#{prefix}", "install"
```

Comment:
I believe the GRDIR needs to be kept, as the `prefix` in homebrew will always be something else than the default prefix the library usually works. This is how it's possible to contain the library into homebrew structure (this can vary for example depending on how the installation is done)

Reply:
We don't need to specify `GRDIR` because `CMAKE_INSTALL_PREFIX` is used as the default value of `GRDIR`.
See also: https://github.com/sciapp/gr/blob/master/CMakeLists.txt#L26-L29

Reply:
Well... Can I remove `GRDIR=#{prefix}` ?

Reply:
I see, it seems that `CMAKE_INSTALL_PREFIX` is defined by the `std_cmake_args`: https://github.com/Homebrew/brew/blob/2dbaaf35aaa4b90e2397019d1e04e26477da9b2e/Library/Homebrew/formula.rb#L1376 so the new code should not require the GRDIR as it being defined to `CMAKE_INSTALL_PREFIX` as @kou mentioned above.

Reply:
all right. I'll put it back.

Reply:
No, it should be removed.

Reply:
Removed!